### PR TITLE
for the lolz

### PR DESCRIPTION
**Pull request by ia-dbuckholz**

### CODE REVIEW

**Review by nrebhun — CHANGES_REQUESTED** [review]
Grr this is not DRY grr

File: someFile.txt [hunk 1]
```diff
@@ -2,3 +2,11 @@ Famous last words:
 
 "I hope this works."
 "This will be great!"
+
+OOOOOOH LOOK CHANGES!
+
```
Comment (nrebhun) [inline someFile.txt:7]: Why is this line blank?!

Comment (asharma90) [inline someFile.txt:7]: lets see the changes..

**Review by asharma90 — COMMENTED** [review]
idk

**Review by nrebhun — CHANGES_REQUESTED** [review]
I'm upset about all this whitespace

**Review by nrebhun — APPROVED** [review]

**Review by nrebhun — CHANGES_REQUESTED** [review]
I changed my mind, this is all terrible. Please change.

File: someOtherFile [hunk 2]
```diff
@@ -0,0 +1 @@
+don't forget to bring your towel
```
Comment (asharma90) [inline someOtherFile:1]: diff try

**Review by nrebhun — APPROVED** [review]
On third thought, this is fine.

File: someFile.txt [hunk 3]
```diff
@@ -2,3 +2,11 @@ Famous last words:
...
+BECAUSE IT CAN BE
+
```
Comment (asharma90) [inline someFile.txt:9]: 💯

**Review by asharma90 — CHANGES_REQUESTED** [review]
tada..

**Review by asharma90 — CHANGES_REQUESTED** [review]
lets try again

**Review by asharma90 — CHANGES_REQUESTED** [review]
i did!!

**Review by asharma90 — CHANGES_REQUESTED** [review]
try try!!

**Pull request merged**